cmd/verify.rb

```diff
@@ -1,79 +1,143 @@
 # typed: true
 # frozen_string_literal: true
 
+require "English"
```

Comment:
I might be missing it, but what here needs `English`?

Reply:
Good catch, I thought `brew style` would catch this, but I'm a ruby noob.

Reply:
Yeah, this is a known deficiency with RuboCop (due to Ruby) -- the language is simply too dynamic to reliably remove unused `require`s 🙂